src/modules/user.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-unused-vars */
+export default async function addNewUser(url, data) {
+  const request = new Request(url, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(data),
+  });
+  // Uncomment to test result
+  const response = await fetch(request);
+}
```

Comment:
- [ ] Please change this function to the arrow function. You can check this. [why you should use the arrow function.](https://www.freecodecamp.org/news/when-and-why-you-should-use-es6-arrow-functions-and-when-you-shouldnt-3d851d7f0b26/)